modules/react/modal/lib/ModalBody.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import {
+  createSubcomponent,
+  ExtractProps,
+  getCanvasTheme,
+  styled,
+  StyledType,
+} from '@workday/canvas-kit-react/common';
+import {Popup} from '@workday/canvas-kit-react/popup';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+import {useModalModel} from './hooks';
+
+export interface ModalHeadingProps extends ExtractProps<typeof Popup.Body, never> {}
```

Comment:
I think this should be ModalHeadingProps.
```export interface ModalBodyProps extends ExtractProps<typeof Popup.Body, never> {}```

Reply:
You'll have to show me how to extend files someday! I tried my hand at it but couldn't get it to work 😅